lib/generate/patient_imports.rb

```diff
@@ -0,0 +1,193 @@
+# frozen_string_literal: true
+
+require "csv"
+
+Faker::Config.locale = "en-GB"
+
+# Use this to generate a cohort import CSV file for performance testing.
+#
+# Usage from the Rails console:
+#
+#     # By default it uses existing locations in the db.
+#     Generate::PatientImports.call
+#
+module Generate
+  class PatientImports
+    attr_reader :ods_code, :organisation, :programme, :urns, :patient_count
+
+    def initialize(
+      ods_code: "A9A5A",
+      programme: "hpv",
+      urns: nil,
+      patient_count: 10
+    )
+      @organisation = Organisation.find_by(ods_code:)
+      @programme = Programme.find_by(type: programme)
+      @urns =
+        urns ||
+          @organisation
+            .locations
+            .select { it.urn.present? }
+            .sample(3)
+            .pluck(:urn)
+      @patient_count = patient_count
+      @nhs_number_prefix = "99"
```

Comment:
Do we need this?

Reply:
Hrm. Ya, this was an old attempt to avoid collisions with existing patient data. I backed it out, but I don't remember why and can't find any hints in the reflog. ¯\_(ツ)_/¯